src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -317,6 +331,16 @@ void CARMACloudPlugin::Broadcast_TCMs()
 			cur_time = 0;	
 			is_started_broadcasting = false;
 		}
+

```

Comment:
Why can't we just update the counter for each req after broadcasting instead of in a separate loop?